projects/map-ol/src/lib/map-ol.service.spec.ts

```diff
@@ -930,15 +930,15 @@ describe('MapOlService State', () => {
 
     const oldZoom = service.getZoom();
     expect(oldZoom).toBeCloseTo(0, 0);
-    const duration = 400;
+    const duration = 250;
     service.zoomInOut('+');
 
     setTimeout(() => {
       const newZoom = service.getZoom();
       expect(newZoom).toBeCloseTo((oldZoom + 1), 0);
       done();
     }, duration);
-  });
+  }, 10000);
```

Comment:
Do we need such a long [timeout](https://jasmine.github.io/api/edge/global.html#it) for this function ?
I only recognized it failing locally.

Reply:
Hi!
Indeed, this test failing locally was the reason I changed the timeout. But since that doesn't seem to be an issue on the server-side, I'll set it back again.

Reply:
For me it failed only sometimes so it was bit wired to check why.